packages/types/src/signature.ts

```diff
@@ -17,6 +17,11 @@ export interface SigningArguments {
 }
 
 export interface RequestSigningArguments extends SigningArguments {
+  /**
+   * The time at which the signed request should no longer be honored.
+   */
+  expiration?: DateInput;
```

Comment:
`SignatureV4.sign()` also uses `RequestSigningArguments`. Adding the parameter here makes signing a request with  a useless `expiration` parameter, it will confuse the users. Can you use another options name(e.g. `RequestPresignArguments`) extending from `RequestSigningArguments` with `expiration` property?

Reply:
Done in https://github.com/aws/aws-sdk-js-v3/pull/1030/commits/50a26586d4a3fc9d3590cee607eb3751c506d115